core/chain_makers.go

```diff
@@ -254,9 +254,9 @@ func makeHeader(chain consensus.ChainReader, parent *types.Block, state *state.S
 	} else {
 		time = parent.Time() + 10 // block time is fixed at 10 seconds
 	}
-
+	root, _ := state.IntermediateRoot(chain.Config().IsEIP158(parent.Number()))
```

Comment:
do we need to check error here

Reply:
I think no need. `chain_makers.go` is usually for testing.